internal/server/data/organization.go

```diff
@@ -13,7 +14,7 @@ type orgCtxKey struct{}
 // OrgFromContext returns the Organization stored using WithOrg.
 func OrgFromContext(ctx context.Context) *models.Organization {
 	org, ok := ctx.Value(orgCtxKey{}).(*models.Organization)
-	if !ok {
+	if !ok || org == nil {
 		// TODO(orgs): panic("no org")
 		return &models.Organization{}
```

Comment:
I definitely don't like that this quietly passes.

Reply:
Ya, this won't stay this way. Tracking it in https://github.com/infrahq/infra/issues/2903. But it lets us make progress without having to fight test failure across multiple packages.